source/agora/network/Manager.d

```diff
@@ -981,7 +981,7 @@ public class NetworkManager
                     headers.map!(header =>
                         tuple(header.height,
                             iota(0, enrolled_validators[header.height]).filter!(i =>
-                                header.validators[i]).count()
+                                header.validators[i] || header.missing_validators.canFind(i)).count() // Take into account the slashed validators
```

Comment:
I think the `i` will be interpreted differently in `validators` and `missing_validators`.

Reply:
Here `i` is the index of the active validator ordered by enrollment utxo. I thought the value in the `missing_validators` array is also the same.

Reply:
`missing_validators` is sorted by UTXO but `enrolled_validator` and `header.validators` are sorted by PublickKey.

Reply:
https://github.com/bosagora/agora/blob/c10b9588a69cf2874e5444d1be58b779c52954f6/source/agora/consensus/state/ValidatorSet.d#L322

Reply:
It was changed to match quite a while ago.

Reply:
Although I see the comment in the code is wrong. I'll add a commit to correct it.

Reply:
I pushed a commit with a correction.